tmpl/button-row.php

```diff
@@ -4,9 +4,19 @@
  *
  * @package Altis-Consent
  */
-$options        = get_option( 'cookie_consent_options' );
+
+use Altis\Consent\Settings;
+
+/**
+ * Get the consent options and set some specific default values.
+ */
+$options        = get_option( 'cookie_consent_options', [
```

Comment:
I would not use an array as a default value for `get_option`. If the option just includes, let's say `banner_message` then `policy_page` and `banner_options` will be missing from the array as `get_option` won't complete the array with the defaults. I would do it this way instead:

```suggestion
$options        = get_option( 'cookie_consent_options', [] );
$options = wp_parse_args( $options, [
      'banner_message' => Settings\get_default_banner_message(),
	'policy_page'    => false,
	'banner-options' => 'none',
] );
```

Reply:
Thanks @igmoweb 
I went back and forth on this because I have a function that will return a single option value from the group, but I didn't want to have to make three separate database calls, and was trying to figure out a good way to handle that. This works for what I was hoping to accomplish. 👍